Task 6/sequelize/sequelize.js

```diff
@@ -0,0 +1,16 @@
+const Sequelize = require('sequelize');
+const UserModel = require('./models/user.js');
+const ProductModel = require('./models/product.js');
+const { configure } = require('sequelize-pg-utilities');
```

Comment:
Don't forget to group your imports by global vs local.
It's a stylistic preference, however it helps tremendously in a larger project.